docs/user_guide/scripting.rst

```diff
@@ -0,0 +1,78 @@
+Scripting
```

Comment:
This is again more implementation focused than user focused. I would suggest the following: Adding Projects with Custom Behaviour